src/test/scala/org/broadinstitute/dsde/firecloud/integrationtest/ResearchPurposeSearchSpec.scala

```diff
@@ -32,16 +32,19 @@ class ResearchPurposeSearchSpec extends FreeSpec with SearchResultValidation wit
     }
 
     "Research purpose for aggregate analysis (NAGR)" - {
-      "should return any dataset where NAGR is false and is (GRU or HMB)" in {
-        val researchPurpose = ResearchPurpose.default.copy(NAGR = true)
-        val searchResponse = searchWithPurpose(researchPurpose)
-        validateResultNames(
-          Set("one", "two", "six", "seven", "eleven", "twelve"),
-          searchResponse
-        )
+      val researchPurpose = ResearchPurpose.default.copy(NAGR = true)
```

Comment:
I extracted the research purpose input and the expected search output in order to run the same tests against the original monolithic search and a simulated external search using the `researchPurposeQuery` function. Ultimately, we may only want one set of these along with some supporting tests to make sure the monolithic search uses the filter code... but maybe not and these tests as they are provide good confidence that both behave the same way.